File: sources/askit_mobile/lib/entity/user_statistics.dart
class UserStatistics {
  final int rankingPoints;
  final int postedQuestions;
  final int postedAnswers;

  UserStatistics(
      {this.rankingPoints, this.postedQuestions, this.postedAnswers});

  factory UserStatistics.fromJson(Map<String, dynamic> json) => UserStatistics(
      rankingPoints: json['rankingPoints'],
      postedQuestions: json['postedQuestions'],
      postedAnswers: json['postedAnswers']);
}

File: sources/askit-mvc/src/main/java/com/project/askit/model/UserStatistics.java
package com.project.askit.model;

public class UserStatistics {

    private Integer postedAnswers;
    private Integer postedQuestions;
    private Integer rankingPoints;

    public UserStatistics() {
    }

    public Integer getPostedAnswers() {
        return postedAnswers;
    }

    public void setPostedAnswers(Integer postedAnswers) {
        this.postedAnswers = postedAnswers;
    }

    public Integer getPostedQuestions() {
        return postedQuestions;
    }

    public void setPostedQuestions(Integer postedQuestions) {
        this.postedQuestions = postedQuestions;
    }

    public Integer getRankingPoints() {
        return rankingPoints;
    }

    public void setRankingPoints(Integer rankingPoints) {
        this.rankingPoints = rankingPoints;
    }

    @Override
    public String toString() {
        return "UserStatistics{" +
                "postedAnswers=" + postedAnswers +
                ", postedQuestions=" + postedQuestions +
                ", rankingPoints=" + rankingPoints +
                '}';
    }
}